docs/static/configuration.asciidoc

```diff
@@ -278,7 +278,7 @@ input { # comments can appear at the end of a line, too
 ----------------------------------
 
 [[event-dependent-configuration]]
-=== Event Dependent Configuration
+=== Accessing Event Data and Fields
```

Comment:
how about `Accessing Event Data and Fields In Configuration`? IMO Configuration is key here..